### ISSUE TITLE
Error in mounted hook: "TypeError: L.GridLayer.GoogleMutant is not a constructor"

### ISSUE THREAD

**Issue by thammarith** [jperelli/vue2-leaflet-googlemutant#11]
I can't get this working with importing from the npm library (`import Vue2LeafletGoogleMutant from 'vue2-leaflet-googlemutant'`) but if I copy the [Vue2LeafletGoogleMutant.vue](https://github.com/jperelli/vue2-leaflet-googlemutant/blob/master/Vue2LeafletGoogleMutant.vue) file and use `import Vue2LeafletGoogleMutant from './Vue2LeafletGoogleMutant'` it works.

When importing using npm, it throws:

```
[Vue warn]: Error in mounted hook: "TypeError: L.GridLayer.GoogleMutant is not a constructor"

found in

---> <VTilelayerGooglemutant>
       <LMap>
         <EditorMapLeaflet> at src/components/maps/editor-map-leaflet.vue
           <MainEditorLeaflet> at src/views/editor/main-editor-leaflet.vue
             <Home> at src/views/home.vue
               <App> at src/app.vue
                 <Root> vue.runtime.esm.js:619
```

```
TypeError: "L.GridLayer.GoogleMutant is not a constructor"
    googleMutant Leaflet.GoogleMutant.js:440
    mounted Vue2LeafletGoogleMutant.js:1
    VueJS 12
    node_modules es6.promise.js:75
    node_modules es6.promise.js:92
    node_modules _microtask.js:18
vue.runtime.esm.js:1888
```

I can't reproduce with the `example.vue`, I can use npm import with the example file but not with mine.

---

## Code

```html
<template>
  <v-map :zoom="12" :center="initialLocation" ref="myMap">
    <v-tilelayer-googlemutant :apikey="apikey" :options="options"></v-tilelayer-googlemutant>
  </v-map>
</template>

<style>
@import url('https://unpkg.com/leaflet@1.5.1/dist/leaflet.css');

html, body {
  height: 100%;
  margin: 0;
}
</style>
```

```javascript
<script>
import L from 'leaflet'
import * as Vue2Leaflet from 'vue2-leaflet'
import Vue2LeafletGoogleMutant from 'vue2-leaflet-googlemutant'
import 'vue2-leaflet-rotatedmarker';

export default {
  components: {
    'v-map': Vue2Leaflet.LMap,
    'v-tilelayer': Vue2Leaflet.LTileLayer,
    'v-tilelayer-googlemutant': Vue2LeafletGoogleMutant
  },
  methods: {
  },
  data () {
    let options = {
      type: 'roadmap'
    }
    return {
      apikey: 'MY_API',
      options,
      initialLocation: L.latLng(13.76, 100.5)
    }
  },
  mounted() {
  }
}
</script>
```

**Comment by jeancaffou** [comment]
Seems to be related to #9, having the same issue

**Comment by thammarith** [comment]
> 
> 
> Seems to be related to #9, having the same issue

Thank you. I googled using the errors, that's probably why it didn't show up.

I'll close this issue once #10 is merged.

**Comment by jeancaffou** [comment]
> I'll close this issue once #10 is merged.

Instead of waiting for the PR merge you can try using it directly in your package.json
Also be aware of your vue2-leaflet version. The fix from #10 didn't work for me until i updated it to ^2.1.1

```
"vue2-leaflet": "^2.1.1",
"vue2-leaflet-googlemutant": "dnasir/vue2-leaflet-googlemutant#dependency-fix",
```

**Comment by thammarith** [comment]
@jeancaffou Thanks, it works!

**Comment by thammarith** [comment]
Fixed in #10
